solutions/LMIO 19-trusis.mdx

```diff
@@ -207,21 +207,29 @@ down and at most $M$ units up?
 <Spoiler title="Spoiler">
 
 Let $b_i$ be as defined above, and let $c_i=M\cdot i+a_i$. Then we should have
-both both
+both $b_{i_j}\le b_{i_{j+1}}$ and $c_{i_j}\le c_{i_{j+1}}$.
+
+So it suffices to find the maximum number of distinct triples of the form
+$(i,b_i,c_i)$ such that
 
 $$
-b_{i_j}\le b_{i_{j+1}}
+(i_1,b_{i_1},c_{i_1})\ll (i_2,b_{i_2},c_{i_2})\ll \cdots \ll (i_k,b_{i_k},c_{i_k}),
 $$
 
-and
+where $(x_1,x_2,x_3)\ll (y_1,y_2,y_3)$ means that $x_j\le y_j$ for $j\in [1,3]$.
+First, let's drop the first element of each triple. The answer remains the same
```

Comment:
Maybe "the answer remains the same as if we kept the first element"? A bit verbose but it's also more explicit.